packages/hierarchy-builder/src/hierarchy-builder/internal/operators/grouping/ClassGrouping.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { HierarchyNode } from "../../../HierarchyNode";
+import { IMetadataProvider } from "../../../Metadata";
+import { getClass } from "../../Common";
+import { GroupingHandlerReturn } from "../Grouping";
+
+interface ClassInfo {
+  fullName: string;
+  name: string;
+  label?: string;
+}
+
+interface ClassGroupingInformation {
+  ungrouped: Array<HierarchyNode>;
+  grouped: Map<string, { class: ClassInfo; groupedNodes: Array<HierarchyNode> }>;
+}
+
+export async function createClassGroups(metadata: IMetadataProvider, nodes: HierarchyNode[]): Promise<GroupingHandlerReturn> {
```

Comment:
please add `@internal` release tags to internal exported apis (I believe docs build requires that and it also makes the intention clear)

Reply:
Added